src/bindings/python/src/openvino/op/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Package: openvino.op
+Low level wrappers for the c++ api in ov::op.
+"""
+
+# flake8: noqa
+
+from openvino._pyopenvino.op import Constant
```

Comment:
Can these be imported directly like in `src/bindings/python/src/openvino/runtime/op/__init__.py`?
`from openvino.op import Constant`

Reply:
I believe here we want to import "base" `Op` directly from `_pyopenvino` while in `openvino/runtime/op/__init__.py` we want to have the same `Op` as in `openvino/op/__init__.py`